internal/preferences_test.go

```diff
@@ -198,6 +229,7 @@ func TestPrefs_SetSameSliceValue(t *testing.T) {
 
 	// We should not fire change when it hasn't changed.
 	for i := 0; i < 2; i++ {
+
```

Comment:
Let's not add new lines outside of the code related to the PR. It is more consistent with our other code to not have a new line here.

Reply:
Done..